### ISSUE TITLE
Add new metadata fields to TEI export

### ISSUE THREAD

**Issue by saracarl** [benwbrum/fromthepage#2120]
We need to add the fields KY requested to the TEI export:

Genre
Date
Place of Creation
Source Location
Source Collection Name
Source Box/Folder
In/Out of Scope
Editorial Notes
The first 7 should be text fields (single lines) the last (Editorial Notes) should be a text box (multi-line)

Depends on #2118 

We should consider how other metadata fields -- like CWRGM's -- fit into this, if at all.
